### PR TITLE
Failure emails

### PR DESCRIPTION
**Pull request by alexflav23**
## Why are you doing this?

This is a 2 in 1 special PR

- Print failure email Trello Card: [**Trello Card**](https://trello.com/c/Jrau1rZr/2271-print-failure-email)
- Guardian weekly failed email: https://trello.com/c/dLmrZO7r/2337-guardian-weekly-failure-email

## Changes

* Added classes for `GuardianWeeklyFailed` and `PrintFailed` emails.
* Added manual tests to check against live Braze deployment/membership-workflow that they work.

### CODE REVIEW

File: support-workers/src/test/scala/com/gu/support/workers/lambdas/FailureHandlerSpec.scala [hunk 1]
```diff
@@ -181,6 +181,49 @@ object DigiPackFailureHandlerManualTest extends Matchers {
         sys.exit(0)
       }
   }
+}
+
+object GuardianWeeklyFailureHandlerTest extends Matchers {
```
Comment (jfsoul) [inline support-workers/src/test/scala/com/gu/support/workers/lambdas/FailureHandlerSpec.scala]: Do you think there’s any unit tests to add as well, or do the existing ones still cover?

Comment (alexflav23) [reply support-workers/src/test/scala/com/gu/support/workers/lambdas/FailureHandlerSpec.scala]: @jfsoul Unit tests are not possible here unfortunately, these are manual checks only. How would you unit that that a specific email box has received an email without writing a mail server? I might be wrong but I don't think that can work.

Comment (jfsoul) [reply support-workers/src/test/scala/com/gu/support/workers/lambdas/FailureHandlerSpec.scala]: Sure, probably not - I was just thinking was there anything to add in `FailureHandlerSpec` class for example, but probably new product failure emails don't change failure handling as such, just which template to trigger.

Comment (alexflav23) [reply support-workers/src/test/scala/com/gu/support/workers/lambdas/FailureHandlerSpec.scala]: @jfsoul Exactly, the way we handle the failure doesn't change, so we have specific objects to trigger sending a specific failure email to an email address. We then look at the sent email and assess if it gets sent and the format is correct. It's fairly manual.

**Pull request merged**